tests/test_sap.py

```diff
@@ -162,6 +162,21 @@ def test_country_code_from_address_country_not_present():
     assert "US" == code
 
 
+def test_parse_invoice_records(mocked_alma, mocked_alma_api_client):
```

Comment:
The order of tests in this module is (or was originally) in the same order as the functions in the module it's testing, to make it easier to find tests for a given function. So I would probably move this up, right after `test_retrieve_sorted_invoices()`.